chat-server/src/services/dataStreamer.ts

```diff
@@ -12,12 +14,15 @@ interface AnswerParams {
   chunks: Content[];
 }
 
-class DataStreamer {
+export interface DataStreamerServiceInterface {
+  answer(params: AnswerParams): Promise<string>;
+}
+export class DataStreamerService {
   // NOTE: for example streaming data, see https://github.com/openai/openai-node/issues/18#issuecomment-1369996933
   async answer({ res, answer, conversation, chunks }: AnswerParams) {
     // TODO: do stuff with the response
```

Comment:
Streaming is out of scope for this PR right?

Reply:
yeah, this was just for tsc